examples/Example/Page/FormSimple.hs

```diff
@@ -46,6 +54,23 @@ formView = do
         checkbox False ~ width 32
         label "Favorite?"
 
+    col ~ gap 5 $ do
+      el $ text "Planet"
+      mapM_ (field "planet")
```

Comment:
This is a misuse of field. It's meant to contain the context of the "name" field of the input. There ought to be a way to use field only once

Reply:
Please see: https://github.com/seanhess/hyperbole/pull/142